crates/rattler_lock/src/lib.rs

```diff
@@ -160,16 +160,16 @@ impl LockedDependency {
     pub fn as_conda(&self) -> Option<&CondaLockedDependency> {
         match &self.kind {
             LockedDependencyKind::Conda(conda) => Some(conda),
-            LockedDependencyKind::Pip(_) => None,
+            LockedDependencyKind::Pypi(_) => None,
         }
     }
 
-    /// Returns a reference to the internal [`PipLockedDependency`] if this instance represents
+    /// Returns a reference to the internal [`PypiLockedDependency`] if this instance represents
     /// a pip package.
```

Comment:
Should these `as_pip` and `is_pip` also change?

Reply:
Yes! Just did that :)